### PR TITLE
Fixed warning when tapping homebrew/core /opt/homebrew/bin is not in your PATH

### PR DESCRIPTION
**Pull request by DenisZackharov**
I add copy if .bashrc and .zshrc not founded and edit .bashrc and .zshrc to fix the installation Homebrew casks.

### CODE REVIEW

File: modules/brew.bash [hunk 1]
```diff
@@ -1,8 +1,23 @@
 #!/usr/bin/env bash
 
-osx_bootstrap="$(cd "$(dirname "$0")/.." && pwd -P)"
```
Comment (DenisZackharov) [inline modules/brew.bash:3]: zsh is not supported this now

File: modules/brew.bash [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+osx_bootstrap="$(cd "$(dirname "$self")/.." && pwd -P)"
```
Comment (DmitryBarskov) [inline modules/brew.bash:3]: You should use `$0` for getting current script name

File: modules/brew.bash [hunk 3]
```diff
@@ -1,8 +1,23 @@
 #!/usr/bin/env bash
 
-osx_bootstrap="$(cd "$(dirname "$0")/.." && pwd -P)"
+osx_bootstrap="$(cd "$(dirname "$self")/.." && pwd -P)" 
 source "$osx_bootstrap/modules/functions.bash"
 
+if [ -f ~/.bashrc ] && [ -f ~/.zshrc ]
```
Comment (DmitryBarskov) [inline modules/brew.bash:6]: It is `.bash_profile` for mac os

File: modules/brew.bash [hunk 4]
```diff
@@ -1,8 +1,23 @@
...
+then
+  chmod 746 ~/.zshrc 
+  chmod 746 ~/.bashrc
+  echo 'export PATH=/opt/homebrew/bin:$PATH' >> ~/.bashrc
+  echo 'export PATH=/opt/homebrew/bin:$PATH' >> ~/.zshrc
+else
```
Comment (DmitryBarskov) [inline modules/brew.bash:12]: What if I have only `.bash_profile` and it's ok for me?

File: modules/brew.bash [hunk 5]
```diff
@@ -1,8 +1,23 @@
...
 
-osx_bootstrap="$(cd "$(dirname "$0")/.." && pwd -P)"
+osx_bootstrap="$(cd "$(dirname "$self")/.." && pwd -P)" 
 source "$osx_bootstrap/modules/functions.bash"
 
+if [ -f ~/.bashrc ] && [ -f ~/.zshrc ]
+then
+  chmod 746 ~/.zshrc 
+  chmod 746 ~/.bashrc
+  echo 'export PATH=/opt/homebrew/bin:$PATH' >> ~/.bashrc
```
Comment (DmitryBarskov) [inline modules/brew.bash:10]: Use `$HOME` variable instead of tilde in scripts.

Comment (DmitryBarskov) [reply modules/brew.bash:10]: Why do you add this to configuration file? Can't you just export this?
```suggestion
export PATH=/opt/homebrew/bin:$PATH
```

File: modules/brew.bash [hunk 6]
```diff
@@ -14,7 +29,7 @@ fi
 info_echo "Install Brew formalue"
 
 brew tap "Homebrew/bundle" 2> /dev/null
-brew bundle install --file="$osx_bootstrap/Brewfile"
+brew bundle install --file="$osx_bootstrap/Brewfile"
```
Comment (DmitryBarskov) [inline modules/brew.bash:32]: remove trailing whitespace

File: modules/brew.bash (hunk 2 above)
Comment (DmitryBarskov) [reply modules/brew.bash:3]: remove trailing whitespace

File: modules/node.bash [hunk 7]
```diff
@@ -8,7 +8,10 @@ info_echo "Enable NVM alias"
 # https://github.com/creationix/nvm/issues/721
 # https://github.com/travis-ci/travis-ci/issues/3854#issuecomment-99492695
 set +e
+#For remove semaphoreci warning
```
Comment (DmitryBarskov) [inline modules/node.bash:11]: It's clear enough even without this comment.
```suggestion

```

**Review by DmitryBarskov — CHANGES_REQUESTED** [review]

File: modules/brew.bash [hunk 8]
```diff
@@ -2,6 +2,7 @@
 
 osx_bootstrap="$(cd "$(dirname "$0")/.." && pwd -P)"
 source "$osx_bootstrap/modules/functions.bash"
+  export PATH="/opt/homebrew/bin:$PATH"
```
Comment (DmitryBarskov) [inline modules/brew.bash:5]:
```suggestion
export PATH="/opt/homebrew/bin:$PATH"
```

**Pull request merged**